internal/pkg/runtime/engines/singularity/data/singularity.conf

```diff
@@ -7,7 +7,8 @@
 # DEFAULT: yes
 # Should we allow users to utilize the setuid program flow within Singularity?
 # note1: This is the default mode, and to utilize all features, this option
-# will need to be enabled.
+# will need to be enabled.  For example, without this option loop mounts of
+# image files will not work; only sandbox images can work.
```

Comment:
Maybe change it a bit:

```
# must be enabled. For example, without this option enabled, loop mounts of
# containers will not work; only will work in sandbox containers.
```

Reply:
@WestleyK I have no problem with changing "will need to be enabled" to "must be enabled" (although that wasn't something I had changed) but the second part doesn't make sense.  @GwendolynK's two alternate wordings make sense, but they are incorrect: loop mounts don't work non-setuid, period.  Sandbox images work without loopmounts.  I also deliberately said "can" instead of "will", because it depends on note2, that is, whether user namespaces are enabled on the system.

Reply:
I could say "only sandbox images can work (subject to note 2)."

Reply:
Or "only sandbox images, which do not use loop mounts, will work (subject to note 2)."

Reply:
Both of those are great alternatives and will fix the issues I saw with the original sentence.